src/Grpc.Net.Client/Balancer/Subchannel.cs

```diff
@@ -261,15 +266,12 @@ private async Task ConnectTransportAsync()
                         SubchannelLog.ConnectBackoffInterrupted(_logger, Id);
 
                         // Delay interrupt was triggered. Reset back-off.
-                        backoffMs = InitialBackOffMs;
+                        backoffPolicy = _manager.BackoffPolicyFactory.Create();
```

Comment:
What's the purpose of recreating the backoff policy here?

Reply:
Ah wait, I think I see what this is for. This will reset the backoff back to the initial values per the requirements in the spec.